include/main.h

```diff
@@ -26,52 +47,84 @@ struct Unk_03006CB0 {
 
 extern struct Unk_020382D0 gUnk_020382D0;
 
+extern u16 gUnk_030023F0;
+extern struct Unk_030023F4 gUnk_030023F4;
 extern u32 gUnk_03002440;
+extern u8 gUnk_03002450[];
 extern u8 gUnk_03002470[];
 extern u16 gUnk_03002480;
 extern u8* gUnk_03002484;
+extern u16 gUnk_03002488;
 extern u8 gUnk_0300248C;
 extern u8 gUnk_03002490[];
+extern u32 gUnk_030024E0;
 extern u8 gUnk_030024E4;
 extern u16 gUnk_030024E8[3];
+extern u8 gUnk_030024F0;
 extern u8 gUnk_03002514;
-extern u16 gUnk_03002520[0x10];
+extern struct Unk_03002520 gUnk_03002520;
+extern u32 gUnk_03002540;
+extern u16 gUnk_03002544;
 extern u8 gUnk_03002548;
+extern u16 gUnk_0300254C;
+extern u8 gUnk_03002550;
 extern u32 gUnk_03002554;
 extern u8 gUnk_03002558;
 extern u8 gUnk_03002760[0x280];
```

Comment:
It's `extern u8 gUnk_03002760[2][0x280];`

Reply:
Maybe, but we don't have confirmation yet

Reply:
Ok, it actually makes sense